### ISSUE TITLE
When enable ivy on version 8.2.2 updateValueAndValidity function doesn't update parent form validity directly.

### ISSUE THREAD

**Issue by chenlevin89** [angular/angular#32204]
After enable ivy, parent form validity doesn't update after child component add new control and call updateValueAndValidity it'a update only after trigger change detection.

@Component({
  selector: 'app-child,
  templateUrl: './child.component.html',
  viewProviders: [
    {provide: ControlContainer, useExisting: FormGroupDirective}
  ]
})
export class ChildComponent implements OnInit {

  form: FormGroup;

  constructor(
    private controlContainer: FormGroupDirective,
    private fb: FormBuilder) {}

  ngOnInit() {
    this.form = this.controlContainer.form;
    this.form.addControl('a', new FormControl(null, Validators.required));
    this.form.updateValueAndValidity();
  }
}

**Comment by kara** [comment]
Closing this issue as it does not follow the issue template. Please file a new issue that follows the template, which gives the team the information needed to investigate.
